build.gradle

```diff
@@ -172,7 +172,7 @@ task assembleTarDistribution(dependsOn: downloadAndInstallJRuby) {
   inputs.files fileTree("${projectDir}/logstash-core/lib")
   inputs.files fileTree("${projectDir}/logstash-core/src")
   inputs.files fileTree("${projectDir}/x-pack")
-  outputs.files file("${buildDir}/logstash-${project.version}.tar.gz")
+  outputs.files file("${buildDir}/logstash-${project.version}-SNAPSHOT.tar.gz")
```

Comment:
minor: Can you conditionally add the SNAPSHOT based on the `version` ?  I understand it doesn't really matter, but it more correct and less surprising if we use this for something else.

Reply:
@jakelandis I don't think I can do it based on the version, but I could mirror what the Ruby side does:

```rb
SNAPSHOT_BUILD = ENV["RELEASE"] != "1"
  PACKAGE_SUFFIX = SNAPSHOT_BUILD ? "-SNAPSHOT" : ""
```

and simply drop the suffix if `System.env()` has `RELEASE` set? WDYT?

Reply:
doh ... for some reason I thought that the `-SNAPSHOT` was in the version.yml. (it's not).  Looking through the versions in this file, this change as-is is consistent.  

If you wan't to update the whole file to honor the RELEASE environment variable, then +1 ... but that is outside scope of this change. Sorry for the noise.

Reply:
@jakelandis right it's a big(ish) change.

Thanks will merge as is and try to move more of the archive building to Gradle soon to get this stuff more consistent. It's kind of crazy the way it is now :(